### ISSUE TITLE
Boss kill not credited.

### ISSUE THREAD

**Issue by Joeeak0324** [dalaranwow/dalaran-wow#4746]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Tonight while running TOC10 for my attunement to ICC10, one tank kicked the other tank on the 4th boss due to him taunting/positioning wrong and taking aggro. This resulted in the loot and the credit for the Twins kill being bugged. We killed all the other bosses and I didn't get the achievement for the attunement. 
**Expected behaviour**: Tell us what should happen instead
The boss should've dropped loot and given the raid credit for the kill. 
**Steps to reproduce the problem**:

1. Go into TOC10
2. Kick someone mid fight 
3. Reinvite them after the fight is over

**Include proofs for this behaviour**
https://imgur.com/a/ZzNQo
The only proof I can provide is a snapshot of my achievements with the TOC10 achievement open with the Twins not being credited. 
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by ashido96** [comment]
Confirming this bug, I was in the raid and no credit (emblems, achievements, etc) was given.
